conformance/setup.go

```diff
@@ -309,7 +309,7 @@ func init() {
 	runPushSetup = true
 	runContentDiscoverySetup = true
 	runContentManagementSetup = true
-	skipEmptyLayerTest = false
+	skipEmptyLayerTest = true
```

Comment:
It looks like this value gets overwritten on line 325. That can be fixed with:

```
	if v, ok := os.LookupEnv(envVarPushEmptyLayer); ok {
		skipEmptyLayerTest, _ = strconv.ParseBool(v)
	}
```

Reply:
Fixed.